pocs/utils/images/__init__.py

```diff
@@ -85,30 +89,47 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
 
 
 def _make_pretty_from_fits(fname, **kwargs):
-    config = load_config()
+    header = getheader(fname)
 
-    title = '{} {}'.format(kwargs.get('title', ''), current_time().isot)
+    # get datetime from fits header and make pretty
+    date_time = header.get('DATE-OBS', '')
+    date_time = date_time.replace('T', ' ', 1)
 
-    new_filename = fname.replace('.fits', '.jpg')
+    title = '{} {}'.format(kwargs.get('title', header.get('FIELD', '')),
+                           date_time)
 
+    new_filename = fname.replace('.fits', '.jpg')
     data = getdata(fname)
-    plt.imshow(data, cmap='cubehelix_r', origin='lower')
-    plt.title(title)
-    plt.savefig(new_filename)
 
-    image_dir = config['directories']['images']
+    percent_value = 99.9
```

Comment:
Just to make it flexible for the future I would pull this from kwargs too. So `kwargs.get('normalize_clip_percent', 99.9)`